okta_rules/okta_mfa_password_reset_audit.yml

```diff
@@ -0,0 +1,22 @@
+AnalysisType: scheduled_query
```

Comment:
not urgent, mainly for my knowledge, why did we write this as a scheduled query and not a py rule?

Reply:
We have a rule for MFA reset in standard, I modified the Okta data model so we could have it monitor Okta logs as well. This is a query that can be used to investigate previous activity